test/11_array.jl

```diff
@@ -32,6 +32,8 @@ for isordered in (false, true)
         @test convert(CategoricalVector{String, DefaultRefType}, x) == x
         @test convert(CategoricalVector{String, UInt8}, x) == x
 
+        @test get(x[2]) == a[2]
```

Comment:
Ah, I always forget how this is organized. I think we have too many small files in that directory. I guess adding it to `convert` would be the most logical choice.